packages/web/package.json

```diff
@@ -41,9 +42,12 @@
     "parse-data-url": "^4.0.1",
     "pathfinding": "^0.4.18",
     "phaser": "^3.55.2",
+    "phaser3-react": "^1.2.6",
+    "phaser3-rex-plugins": "^1.1.65",
     "react": "17.0.2",
     "react-countdown": "^2.3.2",
-    "react-dom": "17.0.2",
+    "react-dom": "^17.0.2",
+    "react-hot-toast": "^2.1.1",
```

Comment:
it it possible to use chakras toast? we can theme it later. would be nice to avoid an additional dep cc @smakosh 
https://chakra-ui.com/docs/feedback/toast

Reply:
Yes, as a matter of fact, I was using chakras toast but they were too limiting in term of the styling / configuration. First of all, chakras toast is limited to only being being "injected" to the body of the html, so we can't display the toasts only in the window of the game, and in terms of styling, we can't really do fancy stuff with it: we can't even change the color of the toast / CSS of the background without needing to use a custom theme